src/app/pages/view/view-records/view-records.component.ts

```diff
@@ -142,33 +159,21 @@ export class ViewRecordsComponent implements OnInit {
         if (event.sortField) {
             params['ordering'] = (event.sortOrder && event.sortOrder < 0) ? '-' + event.sortField : event.sortField;
         }
-        if (this.dateStart) {
-            params['datetime__gte'] = this.dateStart.toISOString();
-        }
-
-        if (this.dateEnd) {
-            params['datetime__lte'] = this.dateEnd.toISOString();
-        }
-
-        if (this.speciesName) {
-            params['record__species_name'] = this.speciesName;
-        }
```

Comment:
This was added by me and it is needed. 
I have the feeling that you didn't pull the latest master.

Reply:
I did pull the latest master down, although I might've accidentally removed this. Anyway, it's back now.